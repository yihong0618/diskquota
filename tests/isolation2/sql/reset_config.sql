!\retcode gpconfig -c diskquota.naptime -v 2;
!\retcode gpstop -u;

1: SHOW diskquota.naptime;
