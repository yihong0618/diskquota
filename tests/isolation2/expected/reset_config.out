!\retcode gpconfig -c diskquota.naptime -v 2;
(exited with code 0)
!\retcode gpstop -u;
(exited with code 0)

1: SHOW diskquota.naptime;
 diskquota.naptime 
-------------------
 2                 
(1 row)
